docs/guide/node-views/react.md

```diff
@@ -80,7 +80,7 @@ import { NodeViewWrapper, NodeViewContent } from '@tiptap/react'
 
 export default () => {
   return (
-    <NodeViewWrapper className="react-component-with-content">
```

Comment:
Intentional?

Reply:
Yeah, renamed to just `className="react-component"` in our demos.